src/runtime/test/particle-api-test.ts

```diff
@@ -93,9 +92,9 @@ describe('particle-api', () => {
     await fooStore.set({id: 'id2', rawData: {value: 'v2'}});
     fooStore._fire = fireFn;
     await fooStore.set({id: 'id3', rawData: {value: 'v3'}});
-    await inspector.verify('update:{"data":{"rawData":{"value":"v1"}}}',
+    await inspector.verify('update:{"data":{"value":"v1"}}',
                            'desync',
-                           'sync:{"rawData":{"value":"v3"}}');
+                           'sync:{"value":"v3"}');
```

Comment:
Discussed offline, this change is due to adding the toJSON method on Entity. It shouldn't affect any other code, just this test (since this test was itself calling JSON.stringify on the entity)